src/containers/shared/css/global.scss

```diff
@@ -152,4 +152,20 @@ a {
   &:hover {
     color: $green-primary;
   }
+
+  &[href^="http://"],
+  &[href^="https://"],
+  &.external {
```

Comment:
is the external class used anywhere? couldnt find it

Reply:
I don't think it is.  I can just remove.